### PR TITLE
Adding pvc and block device notes.

### PR DESCRIPTION
**Pull request by achuzhoy**
The doc needs more clarity - we need to have
a manual PVC created and bound or underlying block devices

Signed-off-by: Alexander Chuzhoy <achuzhoy@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by achuzhoy** [comment]
cc @flaper87 @alknopfler

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc [hunk 1]
```diff
@@ -38,7 +38,26 @@ Installer-provisioned installation of {product-title} requires:
...
+  - kubernetes.io/pvc-protection
+  name: ztp-pvc
+  namespace: edgecluster-deployer
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: standard
```
Comment (flaper87) [inline documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:57]:
```suggestion
```

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc [hunk 2]
```diff
@@ -38,7 +38,26 @@ Installer-provisioned installation of {product-title} requires:
...
     . LSO should be enough but we recommend to use a more reliable storage backend like ODF or NFS in order to avoid issues with the PVs and node scheduling pods.
-- In case you want to use dynamic PV creation using LVMO we've prepared a Tekton task to do that:
+- We need an existing PVC named ztp-pvc to be successfully bound to a PV. You can use the following yaml to create the PVC.
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+  finalizers:
+  - kubernetes.io/pvc-protection
```
Comment (flaper87) [inline documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:48]:
```suggestion
```

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc [hunk 3]
```diff
@@ -50,6 +69,9 @@ run-hub-lvmo-task:
     			--use-param-defaults hub-deploy-lvmo \
     			--showlog
 ```
+    NOTE:: You'll need to make sure that the HUB's master nodes have one available block device to form the required volume group. The total size of these block devices should be no less than 900GB.
```
Comment (flaper87) [inline documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:72]: I would rephrase this as: 

Please, make sure to have an available, default, storageclass configured in the hub cluster for the PVC to be properly bound.

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc [hunk 4]
```diff
@@ -38,7 +38,26 @@ Installer-provisioned installation of {product-title} requires:
     . 1 for the HTTPD server, which will host the RHCOS images.
     . We need to meet the OpenShift Storage requirements for the Hub like (SSD/NVME).
     . LSO should be enough but we recommend to use a more reliable storage backend like ODF or NFS in order to avoid issues with the PVs and node scheduling pods.
-- In case you want to use dynamic PV creation using LVMO we've prepared a Tekton task to do that:
+- We need an existing PVC named ztp-pvc to be successfully bound to a PV. You can use the following yaml to create the PVC.
```
Comment (flaper87) [inline documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:41]:
```suggestion
- Create a PVC called `ztp-pvc` that will be used by the hub pipeline itself. You can use the following yaml to create the PVC.
```

**Review by flaper87 — CHANGES_REQUESTED** [review]

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc (hunk 3 above)
Comment (achuzhoy) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:72]: This note is for the automated PV creation - we need the block devices, right?

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc (hunk 1 above)
Comment (achuzhoy) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:57]: not clear what differs?

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc (hunk 2 above)
Comment (achuzhoy) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:48]: what is different?

Comment (flaper87) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:48]: Remove that section

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc (hunk 1 above)
Comment (flaper87) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:57]: Remove the storage class

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc (hunk 3 above)
Comment (flaper87) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:72]: ah, good point. The diff confused me and I thought it was a generic note. It's correct, then.

**Review by flaper87 — APPROVED** [review]
/lgtm
